src/main/java/edu/unc/lib/boxc/migration/cdm/services/MigrationTypeReportService.java

```diff
@@ -0,0 +1,56 @@
+package edu.unc.lib.boxc.migration.cdm.services;
+
+import edu.unc.lib.boxc.migration.cdm.model.MigrationProject;
+import edu.unc.lib.boxc.migration.cdm.util.PostMigrationReportConstants;
+import org.apache.commons.csv.CSVParser;
+import org.slf4j.Logger;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+/**
+ * Service for counting new migrated objects in box-c (works and files)
+ * @author krwong
+ */
+public class MigrationTypeReportService {
+    private static final Logger log = getLogger(MigrationTypeReportService.class);
+
+    private MigrationProject project;
+
+    /**
+     * @return count new works
+     */
+    public long countWorks() throws IOException {
+        var csvParser = openCsvParser();
+        long numWorks = csvParser.stream()
+                .map(row -> row.get("boxc_obj_type"))
+                .filter(f -> f.toLowerCase().contains("work"))
```

Comment:
The object type column gets populated using this constant
`ResourceType.Work.name()`
So it would make sense to use that when performing the counts as well, in case anything gets renamed.